scss/civicrm/contact/pages/_events.scss

```diff
@@ -1,3 +1,16 @@
+.page-civicrm-event-search {
+  [class*=CRM_Event_Form_Task_] {
+    .crm-form-block > .crm-submit-buttons > .crm-button:first-of-type {
+      margin-left: 0;
+    }
+
+    .status {
+      background-color: $crm-light-green-color;
```

Comment:
can we use $alert-success-bg instead here? which is what we are trying to implement here. would be slightly different for now but, will be covered up sooner